backend/graph/api/episode-resolver.utils.go

```diff
@@ -94,7 +122,7 @@ func (r *episodeResolver) getNextEpisodes(ctx context.Context, episodeID string)
 		}
 		return r.getRelatedEpisodes(ctx, episodeID)
 	}
-	return []int{episodeIDs[index+1]}, nil
+	return []int{next.Current()}, nil
```

Comment:
Can this be next.Keys? On bible kids I need to show a list of the upcoming videos under the video

Reply:
Can expose every next id yes, but that would add additional episodes everywhere. For implementations of auto play the additional episodes are irrelevant.

We might consider making another field for every upcoming episode with pagination.

Reply:
yea, but would make more sense the other way I around: move "nextEpisode" into its own field, since this is an array.
Another approach is to add a limit grahql directive if you only care about the next episode.